classes/Cart.php

```diff
@@ -1024,6 +1024,11 @@ public static function cacheSomeAttributesLists($ipa_list, $id_lang)
         $pa_implode = [];
         $separator = Configuration::get('PS_ATTRIBUTE_ANCHOR_SEPARATOR');
 
+        if ($separator === '-') {
+            // Add a space before the dash between attributes
+            $separator = ' -';
```

Comment:
Why not doing this here https://github.com/PrestaShop/PrestaShop/pull/17423/files#diff-ca7b5e0929fa601d5f167d9d4a6d8e27R1068 ? :thinking: